lib/rules/linebreak-style.js

```diff
@@ -12,20 +13,31 @@
 
 module.exports = function (context) {
     var EXPECTED_LF_MSG = "Expected linebreaks to be 'LF' but found 'CRLF'.",
-        EXPECTED_CRLF_MSG = "Expected linebreaks to be 'CRLF' but found 'LF'.";
+        EXPECTED_CRLF_MSG = "Expected linebreaks to be 'CRLF' but found 'LF'.",
+        MISSING_LF_MSG = "Expected linebreaks to be 'LF' but none were found.",
```

Comment:
I'm not sure this rule should be checking for existence of linebreaks. We have eol-last rules that already checks for this.